### DISCUSSION TITLE
Shared asset net worth calculation

### DISCUSSION THREAD

**Discussion by bakeromso** [firefly-iii/firefly-iii#6229] [General]
Hi all,

Is it possible to take into account on calculating net worth that a shared asset doesnt belong 100% to me? If I share a bank account, I would want that account only to count for 50% in my net worth calculation. 

Ideally, I would love to input expenses on that account as they appear on the bank, but see expenses from that account displayed as 50% of that (so €50 of groceries is shown in recent expenses as €25).

I could manually input every transaction as half its worth of course, but I'm afraid that's too error prone. 

Love to hear what you think!

**Comment by JC5** [comment]
I'm afraid it's not possible to split them like you want in Firefly III.

**Reply by bakeromso** [reply]
That's ok, I will do it manually then. Thank you for the prompt response, it's much appreciated.
